rustls/examples/internal/bogo_shim.rs

```diff
@@ -1041,6 +1041,7 @@ fn main() {
             } else {
                 rustls::ServerSession::new_quic(
                     scfg.as_ref().unwrap(),
+                    1,
```

Comment:
The simplicity of the `1` here seems a little bit misleading. I wonder if we should make this a newtype `QuicVersion` with a `v1` method and maybe `v1_draft(u8)` methods, or something?

Reply:
How about an enum? It's not like we can promise to do the correct thing for future wire-incompatible versions regardless.

Reply:
Yeah, enum sounds good to me. Probably make it non-exhaustive then?